pkg/controller/projects/controller.go

```diff
@@ -319,7 +319,7 @@ func generateCreateProjectOptions(p *v1alpha1.Project) *project.ProjectCreateReq
 	projectCreateRequest := &project.ProjectCreateRequest{
 		Project: &argocdv1alpha1.AppProject{
 			Spec:       projSpec,
-			ObjectMeta: metav1.ObjectMeta{Name: p.Name},
+			ObjectMeta: metav1.ObjectMeta{Name: p.Name, Labels: p.ObjectMeta.Labels},
```

Comment:
I think we should distinguish between MR labels and ones we want to add to the project.

There should be an additional property `spec.forProvider.labels` or `spec.forProvider.projectLabels` where the one can specifiy which to add without mixing them with the labels added to the MR (like Crossplane specific ones).

Reply:
I agree, I wanted to do it like this in the beginning but I did not want to add new fields in the forProvider. I'll make the update ASAP.